docs/modules/ROOT/pages/guides/apollo-federation.adoc

```diff
@@ -61,20 +60,21 @@ const driver = neo4j.driver(
   neo4j.auth.basic("neo4j", process.env.NEO4J_PASSWORD)
 );
 
-const neo4jgraphql = new Neo4jGraphQL({
+const neoSchema = new Neo4jGraphQL({
   typeDefs,
   driver,
 });
 
-neo4jgraphql.getSubgraphSchema().then((schema) => {
-  const server = new ApolloServer({
-    schema,
-  });
+const schema = neoSchema.getSubgraphSchema();
 
-  startStandaloneServer(server, { listen: { port: 4001 } }).then(({ url }) => {
-    console.log(`🚀  Server ready at ${url}`);
-  });
+const server = new ApolloServer({
+  schema,
+  resolvers,
```

Comment:
I probably missed it but where are the `resolvers` defined?

Reply:
Great spot! That was a mistake, it seems :+1:

Reply:
Addressed in https://github.com/neo4j/graphql/pull/3239/commits/b917a0b4183a85b734d4750a00349dbaf9326a7c